pkgs/development/python-modules/camelot/default.nix

```diff
@@ -0,0 +1,64 @@
+{ lib
+, stdenv
+, buildPythonPackage
+, chardet
+, openpyxl
+, charset-normalizer
+, fetchPypi
+, fetchpatch
+, pytest-mock
+, pytest-xdist
+, pytestCheckHook
+, pythonOlder
+, pandas
+, tabulate
+, click
+, pdfminer
+, pypdf
+, opencv3
+}:
+
+buildPythonPackage rec {
+  pname = "camelot-py";
+  version = "0.11.0";
+  format = "setuptools";
+
+  disabled = pythonOlder "3.7";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-l6fZBtaF5AWaSlSaY646UfCrcqPIJlV/hEPGWhGB3+Y=";
+  };
+
+  propagatedBuildInputs = [
+    charset-normalizer
+    chardet
+    pandas
+    tabulate
+    click
+    pdfminer
+    openpyxl
+    pypdf
+    opencv3
+  ];
+
+  nativeCheckInputs = [
```

Comment:
What's the point in having these if you end up disabling the checks/tests a couple of lines below?

Reply:
Sorry, I didn't notice.